projects/lwan/Dockerfile

```diff
@@ -23,3 +23,4 @@ RUN git clone --depth 1 git://github.com/lpereira/lwan
 WORKDIR lwan
 
 COPY build.sh $SRC/
+COPY *.options $SRC/
```

Comment:
I dont think we need this.

Reply:
See this.

Reply:
Ouch. Missed this one. Pushed again without this line.